src/pkg/packager/publish.go

```diff
@@ -141,6 +164,11 @@ type PublishSkeletonOptions struct {
 func PublishSkeleton(ctx context.Context, path string, ref registry.Reference, opts PublishSkeletonOptions) (registry.Reference, error) {
 	l := logger.From(ctx)
 
+	// disallow and sanitize <= 0
+	if opts.Retries < 1 {
```

Comment:
This sounds more like a validation error rather than something we should silently fix for the user.

Reply:
From the SDK perspective this is establishing the default of one for the optional value. When not provided the option is the zero-value `0` - so we haven't been returning an error here. I am inclined to add a debug log here. Thoughts?

Reply:
Added checking for negative values